websites/T/Trackmania.io/presence.ts

```diff
@@ -19,8 +19,7 @@ presence.on("UpdateData", async () => {
   const presenceData: PresenceData = {
     details: "An overview of the Trackmania Live Services.",
     largeImageKey: "logo",
-    state: `Unsupported page (${window.location.hash})`,
-    startTimestamp: Math.floor(Date.now() / 1000)
```

Comment:
why did you deleted it?

Reply:
1) this is useless
2) resets to 0 each time you change page

Reply:
That's how startTimestamp had to be, revert these changes.